src/plugins/expressions/common/expression_functions/specs/theme.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { get } from 'lodash';
+import { ExpressionFunctionDefinition } from '../types';
+
+interface Arguments {
+  variable: string;
+  default: string | number | boolean;
+}
+
+type Output = string | number | boolean | undefined;
+
+export type ExpressionFunctionTheme = ExpressionFunctionDefinition<
+  'theme',
+  null,
+  Arguments,
+  Output
+>;
```

Comment:
This should also get added to the mapping of all function definitions provided by the expressions plugin in `src/plugins/expressions/common/expression_functions/types.ts`

(I'm wondering if we should just get rid of this mapping; it feels like something that will be easy to forget to update?)